src/modules/quizzes.js

```diff
@@ -214,6 +214,57 @@ class Quizzes {
         throw new Error('getQuizResults response data is malformed');
       });
   }
+
+  /**
+   * Retrieves filter expression and recommendation URL from given answers
+   *
+   * @function getQuizResultsConfig
+   * @description Retrieve quiz results page configuration from Constructor.io API
+   * @param {string} quizId - The identifier of the quiz
+   * @param {string} parameters - Additional parameters to refine result set
```

Comment:
Maybe we can drop the `to refine the result set` part and only leave `additional parameters`

Reply:
[ac91e30](https://github.com/Constructor-io/constructorio-client-javascript/pull/288/commits/ac91e30bd1cff82d022b8477d3df6b67f0a673ef)